import_ldraw.py

```diff
@@ -298,18 +306,18 @@ def parse(self, filename):
                     if tmpdate != "":
                         tmpdate = tmpdate.split()
 
-                        # TODO What is this condition for?
-                        # le717 unable to find a case where it is hit.
-                        if (tmpdate[0] == "0" and
-                            len(tmpdate) >= 3 and
-                            tmpdate[1].lower() == "!ldraw_org" and
-                            "part" in tmpdate[2].lower() and
-                            self.part_count > 1
-                        ):
-                            self.subparts.append(
-                                [filename, self.mat, self.colour]
-                            )
-                            break
+                        # LDraw brick comments
```

Comment:
How did my change here break case-insensitivity? What this is a bunch of `if` clauses without any `else` clauses, so I collapsed them into one compound clause and change the strings to lowercase (and added .lower()`) to keep case-insensitivity.